src/components/ActionButton/ActionButton.scss

```diff
@@ -3,6 +3,7 @@
 @import "~vanilla-framework/scss/settings_animations";
 @import "~vanilla-framework/scss/settings_colors";
 @import "~vanilla-framework/scss/patterns_icons";
+@import "~vanilla-framework/scss/base_icon-definitions";
```

Comment:
Have you tried `@import`ing whole vanilla (`/scss/vanilla`)? This should contain all the mixins without producing any CSS output. So you wouldn't depend on things being moved around internally.

Reply:
We used to do that and it caused some issues. We might have to revisit this and see if that's improved.